### ISSUE TITLE
Create Script Link component

### ISSUE THREAD

**Issue by j-pendlebury** [BBC-archive/psammead#2255]
**Is your feature request related to a problem? Please describe.**
We now have services with variants, so we need a script switch.

**Describe the solution you'd like**
Build a component which can be used to switch between variants.

This will be an alpha component.

![image](https://user-images.githubusercontent.com/13763899/65862307-e2560100-e365-11e9-88c7-27d82f9cb3c2.png)

Site | Variant | Link label to open this Variant
-- | -- | --
Serbian | https://www.bbc.com/serbian/lat | Lat
Serbian | https://www.bbc.com/serbian/cyr | Ћир
UK China | https://www.bbc.com/ukchina/simp | 简 (translation for Simp)
UK China | https://www.bbc.com/ukchina/trad | 繁 (translation for Trad)
Zhongwen | https://www.bbc.com/zhongwen/trad | 简 (translation for Simp)
Zhongwen | https://www.bbc.com/zhongwen/simp | 繁 (translation for Trad)

**Testing notes**
[Tester to complete]

- [ ] This feature is expected to need manual testing.

**Additional context**
WSFRONTPGS-100

**Comment by AlistairGempf** [comment]
Have seperate stories for the component by itself, and another integrating it into the brand.

**Comment by AlistairGempf** [comment]
Display site not currently on in component, eg. display "Lat" on Cyrillic Serbian site.

**Comment by ghost** [comment]
@jblaut at 600 and 240px breakpoint have not states defined `hover and focus` what should happen? should we use 320 px.

**Comment by j-pendlebury** [comment]
@sadickisaac sorry, just saw your comment.

Double checked with Fan - hover and focus states should be the same on all breakpoints
